pkg/controller/factory/factory.go

```diff
@@ -54,6 +61,12 @@ func (f *Factory) WithInformers(informers ...Informer) *Factory {
 	return f
 }
 
+// WithPostRunHooks allows to register functions that will run asynchronously after the controller is started via Run command.
+func (f *Factory) WithPostRunHooks(hooks ...PostRunHook) *Factory {
```

Comment:
as commented about. This is not post-run. It is post-start. Pleaes rename.

Reply:
renamed.